snippets/react-snippets.code-snippets

```diff
@@ -135,6 +135,20 @@
         ],
         "description": "Add `role='radio'` to a checkable interactive control. Use radio in place of checkbox if only one item in a group can be checked. Add `aria-checked` to indicate the state of the checkbox."
     },
+    "TabAria": {
+        "prefix": [
+            "TabAria",
+        ],
+        "body": [
+            "role='tab'",
```

Comment:
${{number}} allows users to tab through the parts of the snippets they need to set values for 
For example:
```js
 "role='combobox'",
"aria-haspopup='$1'",
"aria-owns='$2'",
"aria-expanded={$3}",
```
if someone adds the above snippet they will first tab to set $1 -> $2 -> $3 

can you add this for `tab-index` `id` and `aria-selected` instead of hard coding the values -> based on ur really nice description :) it looks like these values depend on the scenario

Reply:
For **tabs**, I think that the recommended `tabIndex` values should only be `'0'` (selected tab) or `'-1'` (not selected). It is possible for the value to be greater than 0 to control focus order but it is considered anti-pattern. Should I put `"tabIndex=${number}"`?  

For **id**, I agree that it should not be hardcoded along with `aria-controls`. Should I put `"id='{tab-id}'"` instead?

For **aria-selected**, I also agree that it should not be hardcoded since it'll depend if tab is currently active and I should put `"aria-selected='{boolean}'"` instead. Does this look good? 

This [site](https://www.w3.org/TR/wai-aria-practices-1.1/examples/tabs/tabs-2/tabs.html) helped me learn more about tab. Learning so much here, thank you Kendall!